code/modules/projectiles/guns/projectile/constructable/blunderbuss.dm

```diff
@@ -97,7 +97,7 @@
 	for(var/i=1, i<=prohibited_items.len, i++)
```

Comment:
That's not even the best way to do it *without* `is_type_in_list()`
BYOND has `for(thing in list)` for a reason